template_scripts/distribution.sh

```diff
@@ -41,6 +41,14 @@ if [ "${DIST#centos}" != "${DIST}" ]; then
     fi
 fi
 
+if [ "$DIST" == "tumbleweed" ] || [ "$DIST" == "leap" ]; then
+    DISTRIBUTION="opensuse"
+    DIST_VER="$DIST"
+
+    YUM_OPTS="$YUM_OPTS --releasever=$DIST_VER --exclude=busybox*"
```

Comment:
```suggestion
    YUM_OPTS+=("--releasever=$DIST_VER" "--exclude=busybox*")
```

Use bash arrays instead of plain strings, but that may require changes elsewhere.